api/provider/aws/test_aws/service_scale_test.go

```diff
@@ -0,0 +1,68 @@
+package test_aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/golang/mock/gomock"
+	provider "github.com/quintilesims/layer0/api/provider/aws"
+	"github.com/quintilesims/layer0/api/tag"
+	awsc "github.com/quintilesims/layer0/common/aws"
+	"github.com/quintilesims/layer0/common/config/mock_config"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func TestServiceScale(t *testing.T) {
```

Comment:
Not sure why you are separating this out - others have just been doing `TestEntityUpdate` and going through each path

Reply:
Yeah, good point. ✅